components/admin_console/plugin_management/plugin_management.jsx

```diff
@@ -871,6 +876,48 @@ export default class PluginManagement extends AdminSettings {
                     >
                         {this.renderEnablePluginsSetting()}
 
+                        {/* PluginSettings should be defined same way as other admin settings. TODO: MM-17871 */}
```

Comment:
nit: 100% support the ticket, but I'd propose we omit the TODO comment here to avoid two sources of truth.

Reply:
👌